### ISSUE TITLE
Can't build on Windows ./mvnw not found

### ISSUE THREAD

**Issue by limonovthesecond2** [cardillan/mindcode#123]
Installed Docker. Cloned this project with git clone. Run in PowerShell at `Mindcode` folder `docker-compose up --build` and get `mvnw: not found` error after `WORKDIR /srv` command. System: Windows 10.
```
[internal] load build definition from Dockerfile
	transferring 550/0 0.007
[internal] load metadata for docker.io/library/eclipse-temurin:17
[internal] load .dockerignore
[internal] load build context
	transferring 66435/0 0.093
[1/5] FROM docker.io/library/eclipse-temurin:17@sha256:9b7aa20276a61013d7f8ce6f5e7b7e7c51a9ec20a9c96f8bd0c677aee2a289fa
[3/5] RUN apt update && apt -y install maven
[2/5] WORKDIR /srv
[5/5] RUN ./mvnw install -Dmaven.test.skip
	/bin/sh: 1: ./mvnw: not found
[4/5] COPY . .
```

**Comment by limonovthesecond2** [comment]
Found a solution to this problem. I needed to change the line ending format from CRLF to LF or run `RUN sed -i 's/\r$//' ./mvnw` before calling `mvnw`. However, testing on Linux is still required

**Comment by limonovthesecond2** [comment]
Fixed in [5ec3dc6](https://github.com/cardillan/mindcode/commit/5ec3dc6f1a0590c8619af9f83262769bf5608ce5)

**Comment by cardillan** [comment]
I had a comment written here I forgot to send, but it looks like that commit has resolved your issue, so it is moot now.
